src/test/java/StepDefinitions/LoginSteps.java

```diff
@@ -0,0 +1,83 @@
+package StepDefinitions;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

For example `pages`

Rename refactorings https://www.jetbrains.com/help/idea/rename-refactorings.html#rename_class_example